### ISSUE TITLE
Add support for IASI L1

### ISSUE THREAD

**Issue by svniemeijer** [stcorp/harp#59]
Add harp ingestion for IASI Level 1 products

This will be a migration of the following BEAT-II ingestions:
- [ ] [IASI_L1](http://www.stcorp.nl/beat/documentation/beatl2-data/index.html#IASI_L1)

The conversion of the spectra from W/m^2.sr.m^-1 to W/cm^2.sr.cm^-1 by dividing by 100 is no longer needed (we use the unit as-is). The same holds for the wavenumber (keep unit m^-1).

This issue depends on #80.

**Comment by svniemeijer** [comment]
This will be a migration of the following BEAT-II ingestions:
- [ ] [IASI_L1](http://www.stcorp.nl/beat/documentation/beatl2-data/index.html#IASI_L1)
